internal/sorted/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright ©2024 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package sorted provides common sorting functions.
+package sorted // import "gonum.org/v1/gonum/internal/ordered"
```

Comment:
Why is the package name changed?

Reply:
See https://github.com/gonum/gonum/pull/1953#discussion_r1536703590